libtransmission/variant-json.cc

```diff
@@ -384,10 +384,7 @@ int tr_jsonParse(char const* source, void const* vbuf, size_t len, tr_variant* s
     data.source = source;
     data.keybuf = evbuffer_new();
     data.strbuf = evbuffer_new();
-    for (int i = 0; i < MAX_DEPTH; ++i)
-    {
-        data.preallocGuess[i] = 0;
-    }
+    std::fill_n(data.preallocGuess, MAX_DEPTH, 0);
```

Comment:
data.preallocGuess = {} is shorter.

Reply:
Hmm. That gives me `array type 'size_t [64]' is not assignable [clang-diagnostic-error]` which makes sense, but I like your way better anyway -- it's shorter _and_ cleaner since it doesn't repeat the sizeof the array.

I changed preallocGuess to be a std::array and then `= {}` worked. Updated in aff2402cbd43b48b1ad9645c281aea1a6b75cd71